bundles/config/org.eclipse.smarthome.config.core.test/src/test/java/org/eclipse/smarthome/config/core/ConfigUtilTest.java

```diff
@@ -38,6 +38,8 @@
     }
 
     private static class L<T> extends ArrayList<T> {
```

Comment:
please remove this inner class completely, use `Collections.singletonList(configDescriptionInteger)` or/and `Arrays.asList(..., ...)` instead.

Also: remove the method 'm' and replace by `Map<String, Object> fooMap = Collections.singletonMap("foo", "1");`.